std/math/fp2.asm

```diff
@@ -166,7 +165,7 @@ mod test {
 
         // Subtract arbitrary elements
         let _ = test_sub(Fp2::Fp2(123, 1234), Fp2::Fp2(567, 5678), Fp2::Fp2(123 - 567, 1234 - 5678));
-        test_sub(Fp2::Fp2(-1, -1), Fp2::Fp2(0x100000000, 1), Fp2::Fp2(-0x100000000 - 1, -2))
+        test_sub(Fp2::Fp2(-1, -1), Fp2::Fp2(2013265920, 1), Fp2::Fp2(-2013265920 - 1, -2))
```

Comment:
Personal preference, but I like hex more here...

Reply:
Same, i'll change it to the hex value